extensions/2.0/Khronos/KHR_lights_punctual/schema/glTF. KHR_lights_punctual.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_lights_punctual glTF extension",
+    "type": "object",
+    "allOf" : [ { "$ref" : "../../../../specification/2.0/schema/glTFChildOfRootProperty.schema.json" } ],
+    "properties": {
+        "lights": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "$ref": "light.schema.json"
+            },
+            "default": []
```

Comment:
To be consistent with the core schemas, an array should have `"minItems": 1` instead of `"default": []`.